client/public/index.html

```diff
@@ -4,13 +4,7 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/LogoGlyphMd.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-
-    <title>HelpGov</title>
+    <title>AskGov</title>
```

Comment:
Couple of things:
- Try to retain some meta description to help crawlers index the page
- [theme-color](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color) might be nice to incorporate as part of the design. Study the linked MDN doc, then consult with @prasanthkumaar as to choice of colours for normal and dark mode
```suggestion
    <meta
      name="description"
      content="Official answers from the Singapore Government"
    />

    <title>AskGov</title>
```

Reply:
Form my conversation with him yesterday, there might be a new theme color in the next design overhaul, but it won't be so soon.

Reply:
Ah okay. Leave it out for now then.